pkg/sriov/sriov_test.go

```diff
@@ -503,6 +503,7 @@ var _ = Describe("Sriov", func() {
 			fakeLink := &FakeLink{netlink.LinkAttrs{}}
 			netconf.HostIFGUID = "01:23:45:67:89:ab:cd:ef"
 
+			mockedNetLinkManger.On("LinkSetName", fakeLink, mock.Anything).Return(nil)
```

Comment:
can you add a UT with the restore name scenario being called ?

Reply:
so if you replace mock.Anything -> netconf.HostIFNames ? it would validate this is being called with the original name as it was "cached" in NetConf